### PR TITLE
villain: add package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Solve https://github.com/BlackArch/blackarch/issues/4207

### CODE REVIEW

File: packages/villain/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+license=('CCBY-NC-ND-4.0')
+depends=('python' 'python-netifaces' 'python-pycryptodome' 'python-pyperclip' 'python-requests' 'python-gnureadline')
+makedepends=('git')
+source=("$pkgname::git+https://github.com/t3l3machus/Villain.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```
Comment (noraj) [inline packages/villain/PKGBUILD:20]: Could you rather use that: https://github.com/BlackArch/blackarch-pkgbuilds/blob/a6f1e0daf6683ff186ac4fbd34fcd024a082c2f4/PKGBUILD-generic-git#L17-L23?
tags are availables

File: packages/villain/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+package() {
+  cd $pkgname
+
+  install -dm 755 "$pkgdir/usr/bin"
+  install -Dm 644 requirements.txt "$pkgdir/usr/share/$pkgname/requirements.txt"
```
Comment (noraj) [inline packages/villain/PKGBUILD:27]: this file is not necessary and also you do a `cp -a` afterward

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/villain/PKGBUILD (hunk 1 above)
Comment (D3vil0p3r) [reply packages/villain/PKGBUILD]: Fixed

File: packages/villain/PKGBUILD (hunk 2 above)
Comment (D3vil0p3r) [reply packages/villain/PKGBUILD]: Fixed

**Review by noraj — COMMENTED** [review]
LGTM by eyes checking, one of use should test it in ba-dev before

**Pull request merged**